README.md

```diff
@@ -63,6 +63,7 @@ Add the following code to your page where appropriate. See the [Branded checkout
     day="<day>"
     default-payment-type="creditCard"
     donor-details="<window variable containing default values for donor's name and contact info>"
+    hide-bank-account-legal-statement="true"
```

Comment:
Let's not add this option to the README for now. FDS wanted it to be obscure so only very specific branded sites they tell us about could use it.

Reply:
Guess I read the helpscout conversation wrong...

> Can you please confirm when you update the branded checkout so it doesn't include EFT disclosure text with references to Cru?

But the rest of it wants the whole thing removed. I'll rework this.